src/open_inwoner/react/modules/Sidenav/Sidenav.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { SideNavigation } from '@gemeente-denhaag/side-navigation'
+
+// Material Icons component
+const MaterialIcon: React.FC<{ name: string }> = ({ name }) => (
+  <span className="material-icons-outlined" aria-hidden="true">
+    {name}
+  </span>
+)
+
+const Sidenav: React.FC = () => {
+  // Get menu data from Django
+  const getMenuData = () => {
```

Comment:
Generic name and type to allow type errors.
```suggestion
  const getJsonScriptData = <T extends unknown>(id: string): T[] => {
```